Modules/Microsoft365DSC/DSCResources/MSFT_EXOReportSubmissionPolicy/MSFT_EXOReportSubmissionPolicy.schema.mof

```diff
@@ -0,0 +1,39 @@
+
+[ClassVersion("1.0.0.0"), FriendlyName("EXOReportSubmissionPolicy")]
+class MSFT_EXOReportSubmissionPolicy : OMI_BaseResource
+{
+    [Key, Description("Specifies the resource is a single instance, the value must be 'Yes'."), ValueMap{"Yes"}, Values{"Yes"}] String IsSingleInstance;
+    [Write, Description("The Identity parameter specifies the report submission policy you want to modify. There can be only one policy.")] String Identity;
```

Comment:
Why have you added this Identity property, when it also has the IsSingleInstance property? Looking at the Set cmdlet documentation, the value of Identity is always "DefaultReportSubmissionPolicy" (see [here](https://learn.microsoft.com/en-us/powershell/module/exchange/set-reportsubmissionpolicy?view=exchange-ps#-identity)). Doesn't it make more sense to populate that specific value in the code instead of asking the user?